modular_citadel/code/modules/client/preferences.dm

```diff
@@ -17,9 +17,9 @@
 	var/autostand = TRUE
 
 	//vore prefs
-	var/toggleeatingnoise = TRUE
-	var/toggledigestionnoise = TRUE
-	var/hound_sleeper = TRUE
```

Comment:
Why this one?

Reply:
That's the voracious sleeper toggle, which is also something that should also be defaulted to off